src/components/onlineTest/StudentForm/BasicDetails/index.jsx

```diff
@@ -278,6 +287,11 @@ const BasicDetails = ({
           <Controller
             control={control}
             name="MiddleName"
+            rules={{
+              required: true,
+              pattern: /^[\p{L}\p{M}]+$/u,
+              maxLength: 10,
```

Comment:
Why are we limiting middle and last names to 10 characters? And also, why would this be different than first names?